grails-app/controllers/org/broadinstitute/orsp/api/FileHelperController.groovy

```diff
@@ -1,10 +1,20 @@
 package org.broadinstitute.orsp.api
 
 import grails.converters.JSON
+import grails.rest.Resource
 import org.broadinstitute.orsp.AuthenticatedController
+import org.broadinstitute.orsp.StorageDocument
 import org.springframework.web.multipart.MultipartFile
 
+import java.text.SimpleDateFormat
+
+@Resource(readOnly = false, formats = ['JSON', 'APPLICATION-MULTIPART'])
 class FileHelperController extends AuthenticatedController{
+    private static final String APPROVED = "Approved"
+    private static final String REJECTED = "Rejected"
+    private static final String PENDING = "Pending"
```

Comment:
I think these strings should all be elements of an `Enum` instead of individual values. You can also add the `Legacy` one as an enum option so old documents can be displayed correctly, if necessary.